docs/using-overseerr/notifications/email.md

```diff
@@ -0,0 +1,39 @@
+# Email
+
+{% hint style="info" %}
+**Media Requested**, **Media Automatically Approved**, and **Media Failed** email notifications are sent to _all_ users with the **Manage Requests** permission, as these notification types are intended for application administrators rather than end users.
+
+**Media Approved**, **Media Declined**, and **Media Available** email notifications are sent to the user who submitted the request.
+{% endhint %}
+
+## Configuration
+
+## Sender Address (required)
+
+Set this to the email address you would like to appear in the "from" field of the email message.
```

Comment:
Add? "Depending on your e-mail provider this may need to be an email address you own. For example, gmail requires this to be your actual e-mail address."